doc/subsystems/conf_s11n/conf_s11n.rst

```diff
@@ -0,0 +1,156 @@
+.. _conf_s11n
+
+Runtime configuration system
+############################
+
+Config subsystem is meant for persisting per-device configuration
+and runtime state for modules.
+
+Configuration items are stored as key-value pairs, where both the key and
```

Comment:
How about:

    Configuration items are stored as key-value pair strings.  By convention,
    the keys can be organized by the package and subtree defining the key,
    for example the key ``id/serial`` would define the ``serial`` configuration
    element for the package ``id``.